docs/config-service.rst

```diff
@@ -62,6 +62,11 @@ All configuration options above in the general ``[gcp]`` may be used here. Addit
 
     For more information on Google Pub/Sub subscriptions, please see `Google's docs on managing subscriptions <subscriptions>`_.
 
+.. option:: max_messages=INT
+
+    `Optional`: Number of Pub/Sub messages to process at a time. Defaults to 25.
```

Comment:
The default I feel is pretty conservative; but we can always update it if we want.

Reply:
Do you know what's the max number of messages Gordon can handle? Is it related to the thread limit?

Reply:
I'm not sure; it can handle 25, but I haven't load tested. It's certainly related to thread limit, but also related to async tasks, too. As I've heard, performance starts to degrade around 50 threaded requests, which I'm not sure translates to async http calls. I feel like 25 is safe to start off at and can turn up from there. What do you think?

Reply:
Yeah, 25 sounds like a good place to start and I think we'll smoke out a better number once we have Gordon running. Thanks for shedding some light on this!